### DISCUSSION TITLE
Vertical Thirds in Landscape

### DISCUSSION THREAD

**Discussion by oaahmad** [rxhanson/Rectangle#972] [Ideas]
I would like to be able to snap to vertical thirds on a landscape monitor. In particular, I want to be able to change the settings so when I drag a window to the bottom of the screen it snaps to the bottom 2/3. I read in other issues that invoking the bottom half command twice does this, but I assume this is only for keyboard shortcuts as that did not work via any UI method (dragging or selecting from the status bar dropdown). I'm generally pretty terrible at remembering keyboard shortcuts, so that's not ideal for me. It's also kind of clunky to have to perform an action twice to get vertical thirds.

**Context:** I have a 42 inch monitor. It's not an ultrawide, so it's very tall. I usually don't like to make my windows the full height of the monitor when I'm working (I keep some files up there for quick access instead).

**Comment by rxhanson** [comment]
Thanks for the feedback. I don't have plans to add any more window sizes/positions to the UI of Rectangle. If you really want to add them in to the code, with the ability to configure via Terminal commands, I see no reason to turn down a good pull request for it. This has been done for a number of other size/positions, like eighths. 

Alternatively, you could use Rectangle Pro (paid), and just use the custom shortcut creator to make them. Here's an example of using it to make top third:
<img width="914" alt="Screenshot 2022-10-30 at 8 41 41 PM" src="https://user-images.githubusercontent.com/13651296/198910339-a732e868-450b-432c-9881-272438ec59a3.png">

Currently, custom shortcuts cannot be configured in an edge snap area in Pro, but I plan to have it in there in an upcoming release. However, custom shortcuts can be used in the window throw feature of pro, which is better than edge snap areas in my opinion anyway.

**Reply by 5knnbdwm** [reply]
Would the custom shortcuts also work with repeating commands. I mainly use "Top Left" "Top Right" "Bottom Left" and "Bottom Right" and repeating changes the width of the windows. 

However on vertical screens I would find it far more beneficial if they would change the height of the windows.

**Reply by rxhanson** [reply]
You can define your own repetitions. Here's a user created tutorial for it: https://github.com/rxhanson/RectanglePro-Community/discussions/5

It's on my backlog to allow additional configuration of a custom shortcut to give it a different size/position on portrait mode screens.
